HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Rocksdb Change Log
 
+## Unlreleased
```

Comment:
Typo: Unreleased
Also need to add other sections, i.e. Public API Change, Bug Fixes. Example:
```
#Rocksdb Change Log
## Unreleased
### New Features
<Your feature>
### Public API Change
### Bug fixes
```

Reply:
Fixed.